2_run/src/run_glm3.R

```diff
@@ -133,7 +133,7 @@ run_glm3_model <- function(sim_dir, nml_objs, model_config, burn_in, burn_out, e
           glm_time <- system.time({glm_code <- GLM3r::run_glm(sim_lake_dir, verbose = FALSE)})[['elapsed']]
           # Pull out the final date from the output to
           # check against requested simulation end date
-          output_dates <- glmtools::get_temp(nc_filepath) %>%
+          output_dates <- glmtools::get_var(nc_filepath, var_name = 'evap') %>%
```

Comment:
Did this change help stop some of the issues with error-ing runs? If so, I would describe or add comments about that here for future us.